src/execution/riscv32/simulator-riscv32.cc

```diff
@@ -2653,11 +2639,7 @@ void Simulator::TraceMemWr(int32_t addr, T value) {
         break;
       case 8:
         if (std::is_integral<T>::value) {
-          SNPrintF(trace_buf_,
-                   "                    (%" PRIu64 ")    int64:%" PRId64
-                   " uint64:%" PRIu64 " --> [addr: %" PRIx32 "]",
-                   icount_, static_cast<int64_t>(value),
-                   static_cast<uint64_t>(value), addr);
+          UNREACHABLE();
```

Comment:
If store double-precision float point （fsd） is traced，the size would be 8

Reply:
`if (std::is_integral<T>::value) ` this is a int value